qutebrowser/html/settings.html

```diff
@@ -37,18 +141,38 @@
   {% for option in configdata.DATA.values()|sort(attribute='name') if not option.no_autoconfig %}
     <tr>
       <!-- FIXME: convert to string properly -->
-      <td class="setting">{{ option.name }} (Current: {{ confget(option.name) | string |truncate(100) }})
+      <td class="setting">{{ option.name }} (Current: <span class="current-setting">{{ confget(option.name) | string |truncate(100) }}</span>)
         {% if option.description %}
-          <p class="option_description">{{ option.description|e }}</p>
+          <p class="option-description">{{ option.description|e }}</p>
         {% endif %}
       </td>
-      <td class="value">
-        <input type="text"
-          id="input-{{ option.name }}"
-          onblur="cset('{{ option.name }}', this.value)"
-          value="{{ confget(option.name) }}">
-        </input>
-      </td>
+      {% if option.typ.valid_values is not none %}
+        <td class="valid-value">
+          {% for value in option.typ.valid_values.values %}
+	    <div class="radio-button">
+	      <input type="radio" id="input-{{ option.name }}-{{ loop.index0 }}"
+	        name="{{ option.name }}" value="{{ value }}"
+	        onclick="cset('{{ option.name }}', this.value)">
+	      <label for="input-{{ option.name }}-{{ loop.index0 }}">
+	        {{ value }}
+	      </label>
+	    </div>
+	    {% if confget(option.name) == value %}
+	      <script type="text/javascript">
+	        checkInput('{{ option.name}}', {{ loop.index0 }})
+	      </script>
```

Comment:
I'm wondering why this JS snippet is needed - couldn't you just add the [`checked` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#attr-checked) directly to the `<input>` element above by doing the `{% if ... %}` inside the `<input>` attributes above?

Reply:
Yep, that's true, I am still getting used to Jinja templates :blush: